data/releases/schedule.yaml

```diff
@@ -64,10 +70,13 @@ schedules:
   maintenanceModeStartDate: 2023-12-28
   endOfLifeDate: 2024-02-28
   next:
-    release: 1.26.11
+    release: 1.26.12
     cherryPickDeadline: 2023-12-01
     targetDate: 2023-12-06
   previousPatches:
+    - release: 1.26.11
+      cherryPickDeadline: N/A
```

Comment:
```suggestion
      cherryPickDeadline: ""
```

Reply:
Sorry, I'd seen N/A for november on the existing page so had assumed it was ok! thanks for fixing :)